packages/frontend/config/build/workers.js

```diff
@@ -5,7 +5,7 @@ const babel = require('rollup-plugin-babel');
 const resolve = require('@rollup/plugin-node-resolve');
 const { terser } = require('rollup-plugin-terser');
 const filesize = require('rollup-plugin-filesize');
-const visualizer = require('rollup-plugin-visualizer');
+// const visualizer = require('rollup-plugin-visualizer');
```

Comment:
If this is ultimately moving to below, I think it would be best to remove the comment. Idk the standard for this sort of thing in ember though, so that's maybe a better question for @NullVoxPopuli